community/examples/Gallery.shtml

```diff
@@ -59,19 +59,18 @@ Well over a decade ago, Nick Kulp already created a program to drive his
 <a href="Panels-Cornwall/Cornwall.shtml">Cornwall RR</a>
 
 <h3>Quaker Valley</h3>
-<a href="http://www.quaker-valley.com/CTC/JMRI_QV_CTC.gif">
-<img src="http://www.quaker-valley.com/CTC/JMRI_QV_CTC.gif" class="floatRight" width="256" height="186" alt="QV panel" />
-</a>
-
-Bob Bucklew built a realistic CTC panel for his
-Quaker Valley Railroad.
+<p>
+Bob Bucklew built a realistic CTC panel for his Quaker Valley Railroad.
 He's created a very nice series of web pages on
-<a href="http://www.quaker-valley.com/CTC/QV_CTCnew.html">how it's done</a>.
+<a href="http://www.quaker-valley.com/CTC/QV_CTCnew.html">how it was done</a>.<br>
+<a href="http://www.quaker-valley.com/CTC/JMRI_QV_CTC.gif">
+    <img src="http://www.quaker-valley.com/CTC/JMRI_QV_CTC.gif" width="256" height="186" alt="QV panel" />
```

Comment:
The gif doesn't display on my browser (Chrome under Windows) although the actual links appear to work.